pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -1417,8 +1412,6 @@ func TestUpdateDefaultLabels(t *testing.T) {
 						v1.LabelZoneRegion:              "new-zone-region",
 						v1.LabelInstanceTypeStable:      "new-instance-type",
 						v1.LabelInstanceType:            "new-instance-type",
-						kubeletapis.LabelOS:             "new-os",
-						kubeletapis.LabelArch:           "new-arch",
```

Comment:
shouldn't these change to the stable OS/Arch labels?

Reply:
done, thanks for reminding me of my mistake